docs/api/allclasses-frame.html

```diff
@@ -0,0 +1,231 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
```

Comment:
These javadoc files are generated by the javadoc compiler, so they should not be checked in